functions.php

```
@@ -366,4 +366,75 @@ function custom_post_labels( $labels ) {
+    
+    $ajax_query = new WP_Query($args);
+    
+    $response = '';
+    if ($ajax_query->have_posts()) {
+        while ($ajax_query->have_posts()) {
+            $ajax_query->the_post();
+            
+            $thumbnail = get_the_post_thumbnail_url(get_the_ID(), 'medium');
+            $url = get_permalink();
+            $title = get_the_title();
+            $date = get_the_date('Y.m.d');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$date'. (undefined)

(UnusedLocalVariable)